src/prasad/FileReading/FreqOfWordInEachLine.java

```diff
@@ -0,0 +1,43 @@
+package prasad.FileReading;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/*Assignment 49 : 3rd Jan'2021
+
+1) Read file and print frequency of "Technocredits" word in each line.*/
+public class FreqOfWordInEachLine {
+	String searchWord = "cars";
+
+	void readFile() {
+		try {
+			Scanner scanner = new Scanner(new File(
+					"C:\\Users\\Prasad Chitale\\Documents\\java_selenium\\Eclipse\\workspace\\Eclipse Java Project\\src\\FileReading\\Text"));
```

Comment:
you should relative path, because this path is not valid on my m/c C\users\Prasad folder.